R/utilities.R

```diff
@@ -509,6 +509,191 @@ ExpMean <- function(x) {
   return(log(x = mean(x = exp(x = x) - 1) + 1))
 }
 
+#' Export Seurat object for UCSC cell browser
+#'
+#' @param object Seurat object
+#' @param dir output directory path
+#' @param dataset.name name of the dataset
+#' @param vars vector of metadata fields to export
+#' @param vars.names list that defines metadata field names after the export.
+#' Should map metadata column name to export name
+#' @param reductions vector of reduction names to export
+#' @param markers.file path to file with marker genes
+#' @param cluster.field name of the metadata field containing cell cluster
+#' @param cb.dir in which dir to create UCSC cell browser content root
+#' @param port \emph{experimental} on which port to run UCSC cell browser after export
+#'
+#' @return ...
```

Comment:
It seems as though this function creates some kind of output (not in R, but on the filesystem). Could you document this somewhere?